uwblueprint/rowan-house
# added progress tags

## Implementation Description
Added Progress Tags in the Course Browsing Page

## Screenshots
<img width="1581" alt="image" src="https://user-images.githubusercontent.com/36215359/191654352-41481aa2-d940-4535-82b4-765db6f873f8.png">


## What should reviewers focus on?
- Ensure that the progress tag matches the progress of the actual course
- Ensure that when a user is not logged in none of the progress tags show up

## Testing Procedure
- Create 3+ courses
- Start at least one of them, complete another, and leave at least one unfinished
- Ensure that when you go to the course view page, they all have the right progress attached to them

## Checklist
